.github/workflows/build-and-test.yml

```diff
@@ -55,7 +60,7 @@ jobs:
             - name: Setup DotNet SDK
               uses: actions/setup-dotnet@v1
               with:
-                  dotnet-version: "3.1.x"
+                  dotnet-version: ${{matrix.options.sdkVersion}}
```

Comment:
I can't figure out from the docs whether you can simply do. [ "2.1.x", "3.1.x" ]

Reply:
Digging through the TypeScript code for the action, it looks to only take string values. Internally, the value is getting passed into here: https://github.com/actions/setup-dotnet/blob/e3d81d94533f1e31c11d94a1fdf6f60f096a7bc5/src/installer.ts#L33

Reply:
Yeah, ran a quick test  in the Web repo. Big fail